src/viking.js

```diff
@@ -1,11 +1,52 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 
 // Viking
-class Viking {}
-
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  // does this work???????????
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+    if (this.health <= 0) {
+      return `${this.name} has died in act of combat`;
+    } else {
+      return `${this.name} has received ${this.damage} points of damage`; //NAMEDAMAGE
```

Comment:
here you're referring to `this.damage` as if it was a property, but it's a parameter of the receiveDamage() method, so it should be used without the "this." 
`return '${this.name} has received ${damage} points of damage'; `